scipy/signal/_peak_finding.py

```diff
@@ -293,13 +308,13 @@ def peak_prominences(vector, peaks, wlen=None):
     Find all peaks and calculate prominences
 
     >>> from scipy import signal
```

Comment:
`from scipy.signal import find_peaks, peak_prominences` to make it a bit more concise and to remove the `signal` repetition